README.md

```diff
@@ -16,8 +16,15 @@ To run this Docker image and keep persistent data in named volumes:
 docker run -d \
   --name omada-controller \
   --restart unless-stopped \
+  -e TZ=Europe/Berlin \
```

Comment:
From an example standpoint, I tend to provide my example with the default values in any environment variables so it would be ideal to provide the example of:

`-e TZ=Etc/UTC`

Reply:
Makes sense, changed accordingly.